2020-03-10 05:05:40,757 INFO 71283 [egg-logrotator] agent logger reload: got log-reload message
2020-03-10 07:25:20,681 ERROR 71283 nodejs.ClusterClientNoResponseError: client no response in 7217096ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 71283
hostname: sunyueyuedeMBP

2020-03-10 09:00:02,643 ERROR 71283 nodejs.ClusterClientNoResponseError: client no response in 3901439ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 71283
hostname: sunyueyuedeMBP

2020-03-10 11:49:48,726 ERROR 71283 nodejs.ClusterClientNoResponseError: client no response in 6585000ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 71283
hostname: sunyueyuedeMBP

2020-03-10 14:03:35,275 ERROR 71283 nodejs.ClusterClientNoResponseError: client no response in 6246069ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 71283
hostname: sunyueyuedeMBP

2020-03-10 15:39:58,450 ERROR 71283 nodejs.ClusterClientNoResponseError: client no response in 5743158ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 71283
hostname: sunyueyuedeMBP

2020-03-10 17:40:52,737 ERROR 71283 nodejs.ClusterClientNoResponseError: client no response in 7214264ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 71283
hostname: sunyueyuedeMBP

2020-03-10 18:24:58,801 ERROR 71283 nodejs.ClusterClientNoResponseError: client no response in 845574ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 71283
hostname: sunyueyuedeMBP

2020-03-10 18:45:22,935 ERROR 71283 nodejs.ClusterClientNoResponseError: client no response in 1204121ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 71283
hostname: sunyueyuedeMBP

2020-03-10 20:46:20,091 ERROR 71283 nodejs.ClusterClientNoResponseError: client no response in 7217138ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 71283
hostname: sunyueyuedeMBP

2020-03-10 23:16:18,004 ERROR 71283 nodejs.ClusterClientNoResponseError: client no response in 7217390ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 71283
hostname: sunyueyuedeMBP

2020-03-11 00:08:42,458 ERROR 71283 nodejs.ClusterClientNoResponseError: client no response in 1343897ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 71283
hostname: sunyueyuedeMBP

2020-03-11 01:27:14,622 ERROR 71283 nodejs.ClusterClientNoResponseError: client no response in 4672142ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 71283
hostname: sunyueyuedeMBP

2020-03-11 03:27:51,760 ERROR 71283 nodejs.ClusterClientNoResponseError: client no response in 7217124ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 71283
hostname: sunyueyuedeMBP

2020-03-11 04:23:37,413 ERROR 71283 nodejs.ClusterClientNoResponseError: client no response in 1545136ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 71283
hostname: sunyueyuedeMBP

2020-03-11 06:24:14,585 ERROR 71283 nodejs.ClusterClientNoResponseError: client no response in 7217155ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 71283
hostname: sunyueyuedeMBP

2020-03-11 07:32:37,311 ERROR 71283 nodejs.ClusterClientNoResponseError: client no response in 2302188ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 71283
hostname: sunyueyuedeMBP

2020-03-11 08:04:41,003 ERROR 71283 nodejs.ClusterClientNoResponseError: client no response in 1903677ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 71283
hostname: sunyueyuedeMBP

2020-03-11 09:00:06,113 ERROR 71283 nodejs.ClusterClientNoResponseError: client no response in 3285084ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 71283
hostname: sunyueyuedeMBP

2020-03-11 12:00:00,005 ERROR 71283 nodejs.ClusterClientNoResponseError: client no response in 7212885ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 71283
hostname: sunyueyuedeMBP

2020-03-11 15:27:18,129 INFO 1290 [egg:logger] init all loggers with options: {"dir":"/Users/hj/Desktop/日常/前端/月月前端/blog/logs/test_vue","encoding":"utf8","env":"local","level":"INFO","consoleLevel":"INFO","disableConsoleAfterReady":false,"outputJSON":false,"buffer":true,"appLogName":"test_vue-web.log","coreLogName":"egg-web.log","agentLogName":"egg-agent.log","errorLogName":"common-error.log","coreLogger":{"consoleLevel":"WARN"},"allowDebugAtProd":false,"type":"agent"}
2020-03-11 15:27:18,141 INFO 1290 [egg:core] dump config after load, 5ms
2020-03-11 15:27:18,184 INFO 1290 [egg-watcher] Start watching: ["/Users/hj/Desktop/日常/前端/月月前端/blog/app","/Users/hj/Desktop/日常/前端/月月前端/blog/config","/Users/hj/Desktop/日常/前端/月月前端/blog/mocks","/Users/hj/Desktop/日常/前端/月月前端/blog/mocks_proxy","/Users/hj/Desktop/日常/前端/月月前端/blog/app.js"]
2020-03-11 15:27:18,184 INFO 1290 [egg-watcher] Start watching: "/Users/hj/Desktop/日常/前端/月月前端/blog/app"
2020-03-11 15:27:18,184 INFO 1290 [egg-watcher] Start watching: "/Users/hj/Desktop/日常/前端/月月前端/blog/config"
2020-03-11 15:27:18,184 INFO 1290 [egg-watcher] Start watching: "/Users/hj/Desktop/日常/前端/月月前端/blog/mocks"
2020-03-11 15:27:18,184 INFO 1290 [egg-watcher] Start watching: "/Users/hj/Desktop/日常/前端/月月前端/blog/mocks_proxy"
2020-03-11 15:27:18,184 INFO 1290 [egg-watcher] Start watching: "/Users/hj/Desktop/日常/前端/月月前端/blog/app.js"
2020-03-11 15:27:18,185 INFO 1290 [egg-watcher:agent] watcher start success
2020-03-11 15:27:18,223 INFO 1290 [egg:core] dump config after ready, 5ms
2020-03-11 15:34:25,960 WARN 1290 [agent:development] reload worker because /Users/hj/Desktop/日常/前端/月月前端/blog/app/service/database.js change
2020-03-11 15:34:25,752 INFO 1290 [egg-watcher] Recieved a change event from eventSource: {"event":"change","path":"/Users/hj/Desktop/日常/前端/月月前端/blog/app/service/database.js","stat":{"dev":16777220,"mode":33188,"nlink":1,"uid":501,"gid":20,"rdev":0,"blksize":4096,"ino":70733619,"size":8618,"blocks":24,"atimeMs":1583912065650.5376,"mtimeMs":1583912065650.9797,"ctimeMs":1583912065650.9797,"birthtimeMs":1575288439844.1033,"atime":"2020-03-11T07:34:25.651Z","mtime":"2020-03-11T07:34:25.651Z","ctime":"2020-03-11T07:34:25.651Z","birthtime":"2019-12-02T12:07:19.844Z"},"remove":false,"isDirectory":false,"isFile":true}
2020-03-11 15:41:53,795 WARN 1290 [agent:development] reload worker because /Users/hj/Desktop/日常/前端/月月前端/blog/app/service/database.js change
2020-03-11 15:41:53,590 INFO 1290 [egg-watcher] Recieved a change event from eventSource: {"event":"change","path":"/Users/hj/Desktop/日常/前端/月月前端/blog/app/service/database.js","stat":{"dev":16777220,"mode":33188,"nlink":1,"uid":501,"gid":20,"rdev":0,"blksize":4096,"ino":70733619,"size":8641,"blocks":24,"atimeMs":1583912513477.6165,"mtimeMs":1583912513478.111,"ctimeMs":1583912513478.111,"birthtimeMs":1575288439844.1033,"atime":"2020-03-11T07:41:53.478Z","mtime":"2020-03-11T07:41:53.478Z","ctime":"2020-03-11T07:41:53.478Z","birthtime":"2019-12-02T12:07:19.844Z"},"remove":false,"isDirectory":false,"isFile":true}
2020-03-11 15:42:49,187 INFO 1290 [egg-watcher] Recieved a change event from eventSource: {"event":"change","path":"/Users/hj/Desktop/日常/前端/月月前端/blog/app/service/database.js","stat":{"dev":16777220,"mode":33188,"nlink":1,"uid":501,"gid":20,"rdev":0,"blksize":4096,"ino":70733619,"size":8650,"blocks":24,"atimeMs":1583912569124.1873,"mtimeMs":1583912569124.488,"ctimeMs":1583912569124.488,"birthtimeMs":1575288439844.1033,"atime":"2020-03-11T07:42:49.124Z","mtime":"2020-03-11T07:42:49.124Z","ctime":"2020-03-11T07:42:49.124Z","birthtime":"2019-12-02T12:07:19.844Z"},"remove":false,"isDirectory":false,"isFile":true}
2020-03-11 15:42:49,394 WARN 1290 [agent:development] reload worker because /Users/hj/Desktop/日常/前端/月月前端/blog/app/service/database.js change
2020-03-11 15:43:18,448 INFO 1290 [egg-watcher] Recieved a change event from eventSource: {"event":"change","path":"/Users/hj/Desktop/日常/前端/月月前端/blog/app/service/database.js","stat":{"dev":16777220,"mode":33188,"nlink":1,"uid":501,"gid":20,"rdev":0,"blksize":4096,"ino":70733619,"size":8647,"blocks":24,"atimeMs":1583912598384.9294,"mtimeMs":1583912598385.4224,"ctimeMs":1583912598385.4224,"birthtimeMs":1575288439844.1033,"atime":"2020-03-11T07:43:18.385Z","mtime":"2020-03-11T07:43:18.385Z","ctime":"2020-03-11T07:43:18.385Z","birthtime":"2019-12-02T12:07:19.844Z"},"remove":false,"isDirectory":false,"isFile":true}
2020-03-11 15:43:18,653 WARN 1290 [agent:development] reload worker because /Users/hj/Desktop/日常/前端/月月前端/blog/app/service/database.js change
2020-03-11 15:43:57,079 WARN 1290 [agent:development] reload worker because /Users/hj/Desktop/日常/前端/月月前端/blog/app/service/database.js change
2020-03-11 15:43:56,878 INFO 1290 [egg-watcher] Recieved a change event from eventSource: {"event":"change","path":"/Users/hj/Desktop/日常/前端/月月前端/blog/app/service/database.js","stat":{"dev":16777220,"mode":33188,"nlink":1,"uid":501,"gid":20,"rdev":0,"blksize":4096,"ino":70733619,"size":8679,"blocks":24,"atimeMs":1583912636824.7405,"mtimeMs":1583912636826.461,"ctimeMs":1583912636826.461,"birthtimeMs":1575288439844.1033,"atime":"2020-03-11T07:43:56.825Z","mtime":"2020-03-11T07:43:56.826Z","ctime":"2020-03-11T07:43:56.826Z","birthtime":"2019-12-02T12:07:19.844Z"},"remove":false,"isDirectory":false,"isFile":true}
2020-03-11 15:45:59,906 WARN 1290 [agent:development] reload worker because /Users/hj/Desktop/日常/前端/月月前端/blog/app/service/database.js change
2020-03-11 15:45:59,703 INFO 1290 [egg-watcher] Recieved a change event from eventSource: {"event":"change","path":"/Users/hj/Desktop/日常/前端/月月前端/blog/app/service/database.js","stat":{"dev":16777220,"mode":33188,"nlink":1,"uid":501,"gid":20,"rdev":0,"blksize":4096,"ino":70733619,"size":8685,"blocks":24,"atimeMs":1583912759637.6833,"mtimeMs":1583912759638.144,"ctimeMs":1583912759638.144,"birthtimeMs":1575288439844.1033,"atime":"2020-03-11T07:45:59.638Z","mtime":"2020-03-11T07:45:59.638Z","ctime":"2020-03-11T07:45:59.638Z","birthtime":"2019-12-02T12:07:19.844Z"},"remove":false,"isDirectory":false,"isFile":true}
2020-03-11 15:46:55,220 WARN 1290 [agent:development] reload worker because /Users/hj/Desktop/日常/前端/月月前端/blog/app/service/database.js change
2020-03-11 15:46:55,012 INFO 1290 [egg-watcher] Recieved a change event from eventSource: {"event":"change","path":"/Users/hj/Desktop/日常/前端/月月前端/blog/app/service/database.js","stat":{"dev":16777220,"mode":33188,"nlink":1,"uid":501,"gid":20,"rdev":0,"blksize":4096,"ino":70733619,"size":8708,"blocks":24,"atimeMs":1583912814911.5925,"mtimeMs":1583912814911.9607,"ctimeMs":1583912814911.9607,"birthtimeMs":1575288439844.1033,"atime":"2020-03-11T07:46:54.912Z","mtime":"2020-03-11T07:46:54.912Z","ctime":"2020-03-11T07:46:54.912Z","birthtime":"2019-12-02T12:07:19.844Z"},"remove":false,"isDirectory":false,"isFile":true}
2020-03-11 15:47:28,107 WARN 1290 [agent:development] reload worker because /Users/hj/Desktop/日常/前端/月月前端/blog/app/service/database.js change
2020-03-11 15:47:27,905 INFO 1290 [egg-watcher] Recieved a change event from eventSource: {"event":"change","path":"/Users/hj/Desktop/日常/前端/月月前端/blog/app/service/database.js","stat":{"dev":16777220,"mode":33188,"nlink":1,"uid":501,"gid":20,"rdev":0,"blksize":4096,"ino":70733619,"size":8711,"blocks":24,"atimeMs":1583912847851.66,"mtimeMs":1583912847852.4575,"ctimeMs":1583912847852.4575,"birthtimeMs":1575288439844.1033,"atime":"2020-03-11T07:47:27.852Z","mtime":"2020-03-11T07:47:27.852Z","ctime":"2020-03-11T07:47:27.852Z","birthtime":"2019-12-02T12:07:19.844Z"},"remove":false,"isDirectory":false,"isFile":true}
2020-03-11 15:48:03,425 WARN 1290 [agent:development] reload worker because /Users/hj/Desktop/日常/前端/月月前端/blog/app/service/database.js change
2020-03-11 15:48:03,216 INFO 1290 [egg-watcher] Recieved a change event from eventSource: {"event":"change","path":"/Users/hj/Desktop/日常/前端/月月前端/blog/app/service/database.js","stat":{"dev":16777220,"mode":33188,"nlink":1,"uid":501,"gid":20,"rdev":0,"blksize":4096,"ino":70733619,"size":8729,"blocks":24,"atimeMs":1583912883153.215,"mtimeMs":1583912883154.0168,"ctimeMs":1583912883154.0168,"birthtimeMs":1575288439844.1033,"atime":"2020-03-11T07:48:03.153Z","mtime":"2020-03-11T07:48:03.154Z","ctime":"2020-03-11T07:48:03.154Z","birthtime":"2019-12-02T12:07:19.844Z"},"remove":false,"isDirectory":false,"isFile":true}
2020-03-11 15:48:34,519 WARN 1290 [agent:development] reload worker because /Users/hj/Desktop/日常/前端/月月前端/blog/app/service/database.js change
2020-03-11 15:48:34,314 INFO 1290 [egg-watcher] Recieved a change event from eventSource: {"event":"change","path":"/Users/hj/Desktop/日常/前端/月月前端/blog/app/service/database.js","stat":{"dev":16777220,"mode":33188,"nlink":1,"uid":501,"gid":20,"rdev":0,"blksize":4096,"ino":70733619,"size":8735,"blocks":24,"atimeMs":1583912914214.8428,"mtimeMs":1583912914215.1855,"ctimeMs":1583912914215.1855,"birthtimeMs":1575288439844.1033,"atime":"2020-03-11T07:48:34.215Z","mtime":"2020-03-11T07:48:34.215Z","ctime":"2020-03-11T07:48:34.215Z","birthtime":"2019-12-02T12:07:19.844Z"},"remove":false,"isDirectory":false,"isFile":true}
2020-03-11 16:11:49,434 WARN 1290 [agent:development] reload worker because /Users/hj/Desktop/日常/前端/月月前端/blog/app/controller/article.js change
2020-03-11 16:11:49,233 INFO 1290 [egg-watcher] Recieved a change event from eventSource: {"event":"change","path":"/Users/hj/Desktop/日常/前端/月月前端/blog/app/controller/article.js","stat":{"dev":16777220,"mode":33188,"nlink":1,"uid":501,"gid":20,"rdev":0,"blksize":4096,"ino":70733612,"size":5163,"blocks":16,"atimeMs":1583914309181.0637,"mtimeMs":1583914309182.2048,"ctimeMs":1583914309182.2048,"birthtimeMs":1574862961939.8423,"atime":"2020-03-11T08:11:49.181Z","mtime":"2020-03-11T08:11:49.182Z","ctime":"2020-03-11T08:11:49.182Z","birthtime":"2019-11-27T13:56:01.940Z"},"remove":false,"isDirectory":false,"isFile":true}
2020-03-11 16:13:04,818 WARN 1290 [agent:development] reload worker because /Users/hj/Desktop/日常/前端/月月前端/blog/app/controller/article.js change
2020-03-11 16:13:04,614 INFO 1290 [egg-watcher] Recieved a change event from eventSource: {"event":"change","path":"/Users/hj/Desktop/日常/前端/月月前端/blog/app/controller/article.js","stat":{"dev":16777220,"mode":33188,"nlink":1,"uid":501,"gid":20,"rdev":0,"blksize":4096,"ino":70733612,"size":5294,"blocks":16,"atimeMs":1583914384559.381,"mtimeMs":1583914384559.7708,"ctimeMs":1583914384559.7708,"birthtimeMs":1574862961939.8423,"atime":"2020-03-11T08:13:04.559Z","mtime":"2020-03-11T08:13:04.560Z","ctime":"2020-03-11T08:13:04.560Z","birthtime":"2019-11-27T13:56:01.940Z"},"remove":false,"isDirectory":false,"isFile":true}
2020-03-11 16:31:50,649 WARN 1290 [agent:development] reload worker because /Users/hj/Desktop/日常/前端/月月前端/blog/app/controller/home.js change
2020-03-11 16:31:50,445 INFO 1290 [egg-watcher] Recieved a change event from eventSource: {"event":"change","path":"/Users/hj/Desktop/日常/前端/月月前端/blog/app/controller/home.js","stat":{"dev":16777220,"mode":33188,"nlink":1,"uid":501,"gid":20,"rdev":0,"blksize":4096,"ino":70733610,"size":1629,"blocks":8,"atimeMs":1583915510380.362,"mtimeMs":1583915510382.1023,"ctimeMs":1583915510382.1023,"birthtimeMs":1583575729074.8442,"atime":"2020-03-11T08:31:50.380Z","mtime":"2020-03-11T08:31:50.382Z","ctime":"2020-03-11T08:31:50.382Z","birthtime":"2020-03-07T10:08:49.075Z"},"remove":false,"isDirectory":false,"isFile":true}
2020-03-11 18:15:23,339 WARN 1290 [agent:development] reload worker because /Users/hj/Desktop/日常/前端/月月前端/blog/app/controller/article.js change
2020-03-11 18:15:23,136 INFO 1290 [egg-watcher] Recieved a change event from eventSource: {"event":"change","path":"/Users/hj/Desktop/日常/前端/月月前端/blog/app/controller/article.js","stat":{"dev":16777220,"mode":33188,"nlink":1,"uid":501,"gid":20,"rdev":0,"blksize":4096,"ino":70733612,"size":5977,"blocks":16,"atimeMs":1583921723072.5012,"mtimeMs":1583921723073.781,"ctimeMs":1583921723073.781,"birthtimeMs":1574862961939.8423,"atime":"2020-03-11T10:15:23.073Z","mtime":"2020-03-11T10:15:23.074Z","ctime":"2020-03-11T10:15:23.074Z","birthtime":"2019-11-27T13:56:01.940Z"},"remove":false,"isDirectory":false,"isFile":true}
2020-03-11 18:15:27,861 WARN 1290 [agent:development] reload worker because /Users/hj/Desktop/日常/前端/月月前端/blog/app/router.js change
2020-03-11 18:15:27,657 INFO 1290 [egg-watcher] Recieved a change event from eventSource: {"event":"change","path":"/Users/hj/Desktop/日常/前端/月月前端/blog/app/router.js","stat":{"dev":16777220,"mode":33188,"nlink":1,"uid":501,"gid":20,"rdev":0,"blksize":4096,"ino":70733616,"size":2170,"blocks":8,"atimeMs":1583921727644.5105,"mtimeMs":1583921727647.5261,"ctimeMs":1583921727647.5261,"birthtimeMs":1583575729078.607,"atime":"2020-03-11T10:15:27.645Z","mtime":"2020-03-11T10:15:27.648Z","ctime":"2020-03-11T10:15:27.648Z","birthtime":"2020-03-07T10:08:49.079Z"},"remove":false,"isDirectory":false,"isFile":true}
2020-03-11 18:19:52,794 WARN 1290 [agent:development] reload worker because /Users/hj/Desktop/日常/前端/月月前端/blog/app/service/database.js change
2020-03-11 18:19:52,587 INFO 1290 [egg-watcher] Recieved a change event from eventSource: {"event":"change","path":"/Users/hj/Desktop/日常/前端/月月前端/blog/app/service/database.js","stat":{"dev":16777220,"mode":33188,"nlink":1,"uid":501,"gid":20,"rdev":0,"blksize":4096,"ino":70733619,"size":8795,"blocks":24,"atimeMs":1583921992493.6528,"mtimeMs":1583921992494.8845,"ctimeMs":1583921992494.8845,"birthtimeMs":1575288439844.1033,"atime":"2020-03-11T10:19:52.494Z","mtime":"2020-03-11T10:19:52.495Z","ctime":"2020-03-11T10:19:52.495Z","birthtime":"2019-12-02T12:07:19.844Z"},"remove":false,"isDirectory":false,"isFile":true}
2020-03-11 18:30:29,834 WARN 1290 [agent:development] reload worker because /Users/hj/Desktop/日常/前端/月月前端/blog/app/controller/article.js change
2020-03-11 18:30:29,630 INFO 1290 [egg-watcher] Recieved a change event from eventSource: {"event":"change","path":"/Users/hj/Desktop/日常/前端/月月前端/blog/app/controller/article.js","stat":{"dev":16777220,"mode":33188,"nlink":1,"uid":501,"gid":20,"rdev":0,"blksize":4096,"ino":70733612,"size":6383,"blocks":16,"atimeMs":1583922629574.7605,"mtimeMs":1583922629578.2827,"ctimeMs":1583922629578.2827,"birthtimeMs":1574862961939.8423,"atime":"2020-03-11T10:30:29.575Z","mtime":"2020-03-11T10:30:29.578Z","ctime":"2020-03-11T10:30:29.578Z","birthtime":"2019-11-27T13:56:01.940Z"},"remove":false,"isDirectory":false,"isFile":true}
2020-03-11 18:35:43,458 WARN 1290 [agent:development] reload worker because /Users/hj/Desktop/日常/前端/月月前端/blog/app/controller/article.js change
2020-03-11 18:35:43,254 INFO 1290 [egg-watcher] Recieved a change event from eventSource: {"event":"change","path":"/Users/hj/Desktop/日常/前端/月月前端/blog/app/controller/article.js","stat":{"dev":16777220,"mode":33188,"nlink":1,"uid":501,"gid":20,"rdev":0,"blksize":4096,"ino":70733612,"size":6770,"blocks":16,"atimeMs":1583922943192.4204,"mtimeMs":1583922943192.8083,"ctimeMs":1583922943192.8083,"birthtimeMs":1574862961939.8423,"atime":"2020-03-11T10:35:43.192Z","mtime":"2020-03-11T10:35:43.193Z","ctime":"2020-03-11T10:35:43.193Z","birthtime":"2019-11-27T13:56:01.940Z"},"remove":false,"isDirectory":false,"isFile":true}
2020-03-11 18:51:23,863 INFO 1290 [egg-watcher] Recieved a change event from eventSource: {"event":"change","path":"/Users/hj/Desktop/日常/前端/月月前端/blog/app/controller/article.js","stat":{"dev":16777220,"mode":33188,"nlink":1,"uid":501,"gid":20,"rdev":0,"blksize":4096,"ino":70733612,"size":6859,"blocks":16,"atimeMs":1583923883748.3538,"mtimeMs":1583923883750.8528,"ctimeMs":1583923883750.8528,"birthtimeMs":1574862961939.8423,"atime":"2020-03-11T10:51:23.748Z","mtime":"2020-03-11T10:51:23.751Z","ctime":"2020-03-11T10:51:23.751Z","birthtime":"2019-11-27T13:56:01.940Z"},"remove":false,"isDirectory":false,"isFile":true}
2020-03-11 18:51:24,066 WARN 1290 [agent:development] reload worker because /Users/hj/Desktop/日常/前端/月月前端/blog/app/controller/article.js change
2020-03-11 19:18:43,659 WARN 1290 [agent:development] reload worker because /Users/hj/Desktop/日常/前端/月月前端/blog/app/controller/article.js change
2020-03-11 19:18:43,457 INFO 1290 [egg-watcher] Recieved a change event from eventSource: {"event":"change","path":"/Users/hj/Desktop/日常/前端/月月前端/blog/app/controller/article.js","stat":{"dev":16777220,"mode":33188,"nlink":1,"uid":501,"gid":20,"rdev":0,"blksize":4096,"ino":70733612,"size":7181,"blocks":16,"atimeMs":1583925523404.1365,"mtimeMs":1583925523404.6033,"ctimeMs":1583925523404.6033,"birthtimeMs":1574862961939.8423,"atime":"2020-03-11T11:18:43.404Z","mtime":"2020-03-11T11:18:43.405Z","ctime":"2020-03-11T11:18:43.405Z","birthtime":"2019-11-27T13:56:01.940Z"},"remove":false,"isDirectory":false,"isFile":true}
2020-03-11 19:25:06,284 WARN 1290 [agent:development] reload worker because /Users/hj/Desktop/日常/前端/月月前端/blog/app/controller/article.js change
2020-03-11 19:25:06,075 INFO 1290 [egg-watcher] Recieved a change event from eventSource: {"event":"change","path":"/Users/hj/Desktop/日常/前端/月月前端/blog/app/controller/article.js","stat":{"dev":16777220,"mode":33188,"nlink":1,"uid":501,"gid":20,"rdev":0,"blksize":4096,"ino":70733612,"size":7590,"blocks":16,"atimeMs":1583925906019.286,"mtimeMs":1583925906021.5493,"ctimeMs":1583925906021.5493,"birthtimeMs":1574862961939.8423,"atime":"2020-03-11T11:25:06.019Z","mtime":"2020-03-11T11:25:06.022Z","ctime":"2020-03-11T11:25:06.022Z","birthtime":"2019-11-27T13:56:01.940Z"},"remove":false,"isDirectory":false,"isFile":true}
2020-03-11 19:46:15,824 WARN 1290 [agent:development] reload worker because /Users/hj/Desktop/日常/前端/月月前端/blog/app/service/database.js change
2020-03-11 19:46:15,618 INFO 1290 [egg-watcher] Recieved a change event from eventSource: {"event":"change","path":"/Users/hj/Desktop/日常/前端/月月前端/blog/app/service/database.js","stat":{"dev":16777220,"mode":33188,"nlink":1,"uid":501,"gid":20,"rdev":0,"blksize":4096,"ino":70733619,"size":8933,"blocks":24,"atimeMs":1583927175554.9338,"mtimeMs":1583927175555.8313,"ctimeMs":1583927175555.8313,"birthtimeMs":1575288439844.1033,"atime":"2020-03-11T11:46:15.555Z","mtime":"2020-03-11T11:46:15.556Z","ctime":"2020-03-11T11:46:15.556Z","birthtime":"2019-12-02T12:07:19.844Z"},"remove":false,"isDirectory":false,"isFile":true}
2020-03-11 19:50:22,044 WARN 1290 [agent:development] reload worker because /Users/hj/Desktop/日常/前端/月月前端/blog/app/service/database.js change
2020-03-11 19:50:21,834 INFO 1290 [egg-watcher] Recieved a change event from eventSource: {"event":"change","path":"/Users/hj/Desktop/日常/前端/月月前端/blog/app/service/database.js","stat":{"dev":16777220,"mode":33188,"nlink":1,"uid":501,"gid":20,"rdev":0,"blksize":4096,"ino":70733619,"size":8940,"blocks":24,"atimeMs":1583927421740.9507,"mtimeMs":1583927421741.6614,"ctimeMs":1583927421741.6614,"birthtimeMs":1575288439844.1033,"atime":"2020-03-11T11:50:21.741Z","mtime":"2020-03-11T11:50:21.742Z","ctime":"2020-03-11T11:50:21.742Z","birthtime":"2019-12-02T12:07:19.844Z"},"remove":false,"isDirectory":false,"isFile":true}
2020-03-11 19:53:33,204 INFO 1290 [egg-watcher] Recieved a change event from eventSource: {"event":"change","path":"/Users/hj/Desktop/日常/前端/月月前端/blog/app/controller/article.js","stat":{"dev":16777220,"mode":33188,"nlink":1,"uid":501,"gid":20,"rdev":0,"blksize":4096,"ino":70733612,"size":8149,"blocks":16,"atimeMs":1583927612881.4824,"mtimeMs":1583927612883.361,"ctimeMs":1583927612883.361,"birthtimeMs":1574862961939.8423,"atime":"2020-03-11T11:53:32.881Z","mtime":"2020-03-11T11:53:32.883Z","ctime":"2020-03-11T11:53:32.883Z","birthtime":"2019-11-27T13:56:01.940Z"},"remove":false,"isDirectory":false,"isFile":true}
2020-03-11 19:53:33,408 WARN 1290 [agent:development] reload worker because /Users/hj/Desktop/日常/前端/月月前端/blog/app/controller/article.js change
2020-03-11 20:05:05,629 INFO 1290 [egg-watcher] Recieved a change event from eventSource: {"event":"change","path":"/Users/hj/Desktop/日常/前端/月月前端/blog/app/controller/category.js","stat":{"dev":16777220,"mode":33188,"nlink":1,"uid":501,"gid":20,"rdev":0,"blksize":4096,"ino":70733613,"size":1230,"blocks":8,"atimeMs":1583928305561.3945,"mtimeMs":1583928305562.8237,"ctimeMs":1583928305562.8237,"birthtimeMs":1578274430792.737,"atime":"2020-03-11T12:05:05.561Z","mtime":"2020-03-11T12:05:05.563Z","ctime":"2020-03-11T12:05:05.563Z","birthtime":"2020-01-06T01:33:50.793Z"},"remove":false,"isDirectory":false,"isFile":true}
2020-03-11 20:05:05,834 WARN 1290 [agent:development] reload worker because /Users/hj/Desktop/日常/前端/月月前端/blog/app/controller/category.js change
2020-03-11 23:35:12,636 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 706301ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-11 23:56:12,802 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 1220148ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 00:13:26,003 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 993180ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 00:48:48,135 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 2102115ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 01:09:47,320 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 1219172ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 01:30:46,532 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 1219204ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 01:51:27,003 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 1220454ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 02:12:26,225 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 1219203ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 02:15:36,403 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 150157ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 02:17:59,587 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 103164ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 02:38:38,776 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 1219171ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 02:59:37,961 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 1219163ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 03:20:36,996 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 1219005ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 03:36:10,318 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 913304ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 03:48:41,117 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 710787ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 03:58:58,303 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 577173ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 04:08:56,003 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 577687ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 04:19:13,200 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 577176ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 04:29:29,415 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 576192ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 04:39:46,615 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 577177ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 04:49:43,812 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 577188ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 05:00:00,997 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 577166ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 05:10:18,197 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 577177ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 05:20:15,372 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 577165ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 05:30:32,587 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 577198ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 05:40:49,053 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 576448ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 05:50:46,248 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 577178ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 06:01:03,425 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 577159ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 06:11:20,420 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 576976ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 06:22:45,790 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 577184ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 06:27:08,996 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 223186ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 06:28:59,199 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 70172ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 06:38:56,400 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 577192ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 06:44:21,723 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 285309ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 06:55:42,123 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 577193ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 07:05:58,748 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 576609ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 07:14:10,939 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 452165ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 07:24:09,003 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 578055ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 07:34:26,201 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 577180ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 07:44:42,392 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 576171ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 07:54:59,908 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 577496ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 08:04:57,094 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 577168ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 08:15:14,286 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 577173ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 08:25:30,466 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 576163ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 08:35:27,667 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 577187ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

2020-03-12 08:45:08,840 ERROR 1290 nodejs.ClusterClientNoResponseError: client no response in 541153ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/hj/Desktop/日常/前端/月月前端/blog/node_modules/cluster-client/lib/leader.js:77:23)
    at ontimeout (timers.js:427:11)
    at tryOnTimeout (timers.js:289:5)
    at listOnTimeout (timers.js:252:5)
    at Timer.processTimers (timers.js:212:10)
name: "ClusterClientNoResponseError"
pid: 1290
hostname: bogon

